packages/account/src/Components/form-footer/form-footer.jsx

```diff
@@ -1,3 +1,23 @@
 import React from 'react';
+import { PropTypes } from 'prop-types';
+import { isMobile, PlatformContext } from '@deriv/shared';
+import classNames from 'classnames';
 
-export const FormFooter = ({ children }) => <div className='account-form__footer'>{children}</div>;
+const FormFooter = ({ children }) => {
+    const { is_dashboard } = React.useContext(PlatformContext);
+    return (
+        <div
+            className={classNames('account-form__footer', {
+                'account-form__footer--dashboard': is_dashboard && !isMobile(),
```

Comment:
How about using `isDesktop` also from `@deriv/shared` instead of `!isMobile`?

Reply:
It was temporary. As responsive design updated this week all !isMobile() are removed